notify-service/notify-api/src/notify_api/__init__.py

```diff
@@ -32,6 +34,8 @@
 
 logger = StructuredLogging.get_logger()
 
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "./c4hnrd-dev-d8e2a3f7bd86.json"
```

Comment:
do we need the token? based on what we talked last time i was under impression this only needed if connecting from outside of GCP, i.e. could use iam to give access to cloudsql instance. I assume notify-api runs in gcp.

Reply:
i see you removed it already